rocketpy/__init__.py

```diff
@@ -35,3 +35,4 @@
     TrapezoidalFins,
 )
 from .simulation import Flight
+from .plots.compare import Compare, CompareFlights
```

Comment:
I'm not a big fan of this code line. I think restricting the user to access the Compare class from only the `plots` module would allow for a cleaner `rocketpy.__init__.py` file.

However, it is a matter of preference since we don't have specific rules for that. So go ahead and merge it. 

Just keep in mind that *it is not mandatory* to expose all the classes to the main __init__.py file

Reply:
I also prefer accessing it only through the plots module, but the way it is done is also fine and not a deal breaker for this PR. I believe it is more a matter of preference/style than correctness.